server/endpoints/hook.go

```diff
@@ -232,3 +226,18 @@ func Hook(db *sqlx.DB, secret string) echo.HandlerFunc {
 		return c.String(http.StatusOK, msg)
 	}
 }
+
+func shouldUpdatePR(logger *logrus.Entry, ghClient *gh.Client, repo *github.Repository, pr *github.PullRequest) (bool, error) {
+	updateLabel, err := ghClient.HasLabels(pr, config.UpdateMe)
+	if err != nil {
+		return false, errors.Wrapf(err, "cannot check if %s-%s has update label", repo.GetFullName(), pr.GetNumber())
```

Comment:
Change the format specifier to `%s-%d` `pr.GetNumber` returns an int https://github.com/google/go-github/blob/master/github/github-accessors.go#L5967.